exp/controllers/awsmachinepool_controller.go

```diff
@@ -371,7 +370,7 @@ func (r *AWSMachinePoolReconciler) createPool(machinePoolScope *scope.MachinePoo
 
 func (r *AWSMachinePoolReconciler) findASG(machinePoolScope *scope.MachinePoolScope, asgsvc services.ASGInterface) (*expinfrav1.AutoScalingGroup, error) {
 	// Query the instance using tags.
-	asg, err := asgsvc.GetASGByName(machinePoolScope)
```

Comment:
Can we delete this method from Mock (and interface)?

Reply:
done